rpgpy/nc.py

```diff
@@ -152,3 +188,21 @@ def _get_measurement_date(file: netCDF4.Dataset) -> list:
     date = utils.rpg_seconds2date(ma.min(time), date_only=True)
     assert_array_equal(date, utils.rpg_seconds2date(ma.max(time), date_only=True))
     return date
+
+def _generator_files(dir: str = cwd):
```

Comment:
Here I didn't the return value because I wasn't 100% sure that it's `None` because technically it returns a generator when the function is called.. but being a function for internal use it's maybe not too relevant!